.babelrc

```diff
@@ -0,0 +1,6 @@
+{
+    "presets": [
+      "@babel/preset-env"
+    ]
+  }
+  
```

Comment:
Please remove the extra line. There should only be a single newline at the end of every file